src/renderer/components/dashboard/Holdings.vue

```diff
@@ -20,20 +19,16 @@ export default {
       });
     },
 
-    ...mapGetters([
-      'sendInProgress',
-    ]),
```

Comment:
This should work...

https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

Reply:
I will restore this; didn't mean to remove.

Reply:
Oh, actually I had switched to the other syntax because I was going to add something, but then changed my mind and when I removed it didn't go back to the original syntax.